proto/prysm/v1alpha1/validator.proto

```diff
@@ -136,12 +136,29 @@ service BeaconNodeValidator {
         };
     }
 
+    // DEPRECATED: do not use.
+    rpc GetBlock(BlockRequest) returns (BeaconBlock) {
+        option deprecated = true;
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/block"
+        };
+    }
+
+    // DEPRECATED: do not use.
+    rpc ProposeBlock(SignedBeaconBlock) returns (ProposeResponse) {
+        option deprecated = true;
+        option (google.api.http) = {
+            post: "/eth/v1alpha1/validator/block"
+            body: "*"
+        };
+    }
+
     // Retrieves the latest valid beacon block to be proposed on the beacon chain.
     //
     // The server returns a new beacon block, without proposer signature, that can be
     // proposed on the beacon chain. The block should be filled with all the necessary
     // data for proposer to sign.
-    rpc GetBlock(BlockRequest) returns (BeaconBlock) {
+    rpc GetBeaconBlock(BlockRequest) returns (GenericBeaconBlock) {
         option (google.api.http) = {
             get: "/eth/v1alpha1/validator/block"
```

Comment:
I think gRPC gateway is going to have a hard time when an endpoints match multiple service methods.

Reply:
Let's name the new one `/beacon-block` or `/beacon_block`, following the current convention.

Reply:
shouldn't it be more intuitive to call it v1alpha2/validator/block? Only reason I called it GetBeaconBlock was to avoid naming conflicts in this package. When we move it into the v1alpha2 namespace we will likely keep t as GetBlock